x-pack/plugin/sql/src/main/java/org/elasticsearch/xpack/sql/expression/function/FunctionRegistry.java

```diff
@@ -413,4 +436,8 @@ private static String normalize(String name) {
         // translate CamelCase to camel_case
         return StringUtils.camelCaseToUnderscore(name);
     }
+    
+    protected static final TreeSet<FunctionDefinition> buildSetOfFunctions(FunctionDefinition... defs) {
```

Comment:
Returning a `List` instead of the `TreeSet` so it can be concatenated into the list makes things easier - the consumer of the list doesn't care about the ordering.